Sources/Controllers/2.0/OAQuickActionHudViewController.mm

```diff
@@ -55,9 +58,13 @@ - (void)viewDidLoad
 {
     [super viewDidLoad];
     
-    [_quickActionFloatingButton setImage:[UIImage templateImageNamed:@"ic_custom_quick_action"]
-                                forState:UIControlStateNormal];
-    _quickActionFloatingButton.hidden = ![_settings.quickActionIsOn get];
+    _quickActionFloatingButton.alpha = [_settings.quickActionIsOn get] ? 1 : 0;
+    _quickActionFloatingButton.userInteractionEnabled = [_settings.quickActionIsOn get];
```

Comment:
Test is the area behind the hidden quick action button can be used for map interaction.

Reply:
https://www.dropbox.com/s/zw9kedqfffl6tc7/Screen%20Recording%202021-08-05%20at%2015.38.58.mov?dl=0